submissions/Harnytskyi/memory_pair_game/script.js

```diff
@@ -1,51 +1,40 @@
-const CARDS = ["Kyiv", "Kharkiv", "Lviv", "Odesa", "Dnipro", "Poltava", "Zhytomyr", "Khmelnytskyi", "Luhansk", "Rivne", "Cherkasy", "Mykolaiv", "Simferopol", "Zaporizhzhia"];
+const CITIES = ["Kyiv", "Kharkiv", "Lviv", "Odesa", "Dnipro", "Poltava", "Zhytomyr", "Khmelnytskyi", "Luhansk", "Rivne", "Cherkasy", "Mykolaiv", "Simferopol", "Zaporizhzhia"];
 const GRID = document.getElementById("allCards");
-const FINISH_MATCH_CARDS = 8;
+const quantityCardPairs = 8;
 const DELAY = 800;
-let matchCards = 0;
+let matchedCards = 0;
 let firstCard;
 
-function choseCards(cardArray) {
-    let randomCards = cardArray.sort(() => 0.5 - Math.random());
-    randomCards = randomCards.slice(0, 8)
+function chooseCardsRandomly(cards) {
+    let randomCards = cards.sort(() => 0.5 - Math.random());
+    randomCards = randomCards.slice(0, quantityCardPairs)
     randomCards = randomCards.concat(randomCards);
     randomCards = randomCards.sort(() => 0.5 - Math.random())
     return randomCards;
 }
 
-function displayCard() {
-    const ALL_CARDS = document.createDocumentFragment();
+function displayCards() {
     GRID.innerHTML = "";
-    let CHOSEN_CARDS = choseCards(CARDS);
+    let CHOSEN_CARDS = chooseCardsRandomly(CITIES);
```

Comment:
why `let` here ?

Reply:
I created a variable because I didn't notice that it is assigned only once. 
Now I realized that `const` would be enough here